packages/rainbowkit/src/components/RainbowKitProvider/RainbowKitProvider.tsx

```diff
@@ -111,42 +120,46 @@ export function RainbowKitProvider({
             value={showRecentTransactions}
           >
             <TransactionStoreProvider>
-              <AvatarContext.Provider value={avatarContext}>
-                <AppContext.Provider value={appContext}>
-                  <ThemeIdContext.Provider value={id}>
-                    <ModalProvider>
-                      {theme ? (
-                        <div {...createThemeRootProps(id)}>
-                          <style
-                            // eslint-disable-next-line react/no-danger
-                            dangerouslySetInnerHTML={{
-                              // Selectors are sanitized to only contain alphanumeric
-                              // and underscore characters. Theme values generated by
-                              // cssStringFromTheme are sanitized, removing
-                              // characters that terminate values / HTML tags.
-                              __html: [
-                                `${selector}{${cssStringFromTheme(
-                                  'lightMode' in theme ? theme.lightMode : theme
-                                )}}`,
-
-                                'darkMode' in theme
-                                  ? `@media(prefers-color-scheme:dark){${selector}{${cssStringFromTheme(
-                                      theme.darkMode,
-                                      { extends: theme.lightMode }
-                                    )}}}`
-                                  : null,
-                              ].join(''),
-                            }}
-                          />
-                          {children}
-                        </div>
-                      ) : (
-                        children
-                      )}
-                    </ModalProvider>
-                  </ThemeIdContext.Provider>
-                </AppContext.Provider>
-              </AvatarContext.Provider>
+              <AvatarFallbackContext.Provider value={avatarFallbackContext}>
```

Comment:
Would like to revert the addition of `AvatarFallbackContext` and bundle context into `AvatarContext`